apps/charterafrica/src/lib/tools/models.js

```diff
@@ -0,0 +1,99 @@
+import api from "../payload";
+
+export const ORGANIZATION_COLLECTION = "organisations";
+export const PEOPLE_COLLECTION = "people";
+export const TOOL_COLLECTION = "tool";
```

Comment:
```suggestion
export const TOOLS_COLLECTION = "tools";
```
Aren't collection in plural?